erpnext/manufacturing/doctype/job_card/job_card.js

```diff
@@ -73,7 +73,24 @@ frappe.ui.form.on('Job Card', {
 		if (frm.doc.docstatus == 0 && !frm.is_new() &&
 			(frm.doc.for_quantity > frm.doc.total_completed_qty || !frm.doc.for_quantity)
 			&& (frm.doc.items || !frm.doc.items.length || frm.doc.for_quantity == frm.doc.transferred_qty)) {
-			frm.trigger("prepare_timer_buttons");
+
+			//if Job Card is link to Work Order, the job card must not be able to start if Work Order not "Started"
+			// and if stock mvt for WIP is required
+			if (frm.doc.work_order) {
+				frappe.db.get_value('Work Order', frm.doc.work_order, 'skip_transfer').then((r) => {
+					if (r.message.skip_transfer !== 1 ) {
```

Comment:
The last else can be eliminated by combining both if conditions.
Also does "Completed" work order need a timer button?
```suggestion
					if (result.skip_transfer !== 1  && result.status == 'In Process') {
					     frm.trigger("prepare_timer_buttons");
					}
```